ramls/loan.json

```diff
@@ -53,6 +53,11 @@
       "description": "ID of last policy used in relation to this loan",
       "type": "string"
     },
+    "loanPolicyName": {
```

Comment:
The module descriptor and RAML documentation need updating with a new minor interface version to reflect the additional property

Reply:
A while ago, it was discussed whether to go with a separate object property e.g. `loanPolicy`, what led to the decision of using `loanPolicyName`?

Reply:
Resolved